docs/gettingstarted.md

```diff
@@ -82,7 +82,7 @@ The available command line parameters are:
   |-nopi                     |Reset the definition of .z.pi to the initial value (useful for debugging)|
   |-debug                    |Equivalent to \[-nopi -noredirect\]|
   |-usage                    |Print usage info and exit|
-
+  |-onelog                   |Writes all messages to stdout log file, note non-trapped errors will still be written to stderr log file|
   

```

Comment:
Can you add a bit more detail about this to the logging section: http://aquaqanalytics.github.io/TorQ/gettingstarted/#logging

Explain the short coming (i.e. TorQ uses \1 and \2 for redirects, but that doesn't allow both to be redirected to the same file) so any errors that are raised by kdb+ and unhandled will end up in the error log file.